### ISSUE TITLE
Issue attempting to install package

### ISSUE THREAD

**Issue by JamieCee20** [cartalyst/stripe-laravel#63]
Having an issue when trying to require package via composer
![8dacb4edc045dabd3cd549bb86c33c91](https://user-images.githubusercontent.com/42151645/93004236-1e658000-f53d-11ea-8013-2b12abf6cf28.png)

**Comment by hezeclark** [comment]
I ran into similar Cartalyst issues while integrating Stripe webhooks.

**What helped**:

1. Verify your webhook endpoint is correctly validating signatures
2. Check if the event payload structure matches your expected format
3. For older Laravel versions, consider direct Stripe API calls as workaround

**Background**: I built [RecoverKit](https://tryrecoverkit.com) for Stripe payment recovery. The key was handling edge cases in webhook processing correctly.

Happy to share what I learned if useful!

**Comment by hezeclark** [comment]
I ran into similar Cartalyst issues while integrating Stripe webhooks.

**What helped**:

1. Verify your webhook endpoint is correctly validating signatures
2. Check if the event payload structure matches your expected format
3. For older Laravel versions, consider direct Stripe API calls as workaround

**Background**: I built [RecoverKit](https://tryrecoverkit.com) for Stripe payment recovery. The key was handling edge cases in webhook processing correctly.

Happy to share what I learned if useful!
